gradle/libs.versions.toml

```diff
@@ -86,6 +86,7 @@ truth = "com.google.truth:truth:1.0"
 hamcrest = { module = "org.hamcrest:hamcrest", version.ref = "hamcrest" }
 hamcrestLibrary = { module = "org.hamcrest:hamcrest-library", version.ref = "hamcrest" }
 jsonPathAssert = "com.jayway.jsonpath:json-path-assert:2.4.0"
+jsonAssert = "org.skyscreamer:jsonassert:1.5.0"
```

Comment:
https://github.com/skyscreamer/JSONassert 

allows us to compare json contents, without exact mathcing of all spaces and newlines